lib/Analysis/Utility.cpp

```diff
@@ -58,11 +58,6 @@ SmallVector<SmallVector<unsigned>> ReduceOpHelper::getScratchConfigsFast() {
   unsigned numWarps = triton::gpu::TritonGPUDialect::getNumWarps(mod);
```

Comment:
I noticed there's still a FIXME here. Are you also going to fix this issue?

Reply:
Yes. At the moment it is correct but uses a bit more shared memory. Fixing this is not much work, but I have to be very careful because it might cause outbounded shared memory access. I plan to do this later as I have some higher priority work to do now. This does not affect the performance much.

Reply:
OK, as long as there's a note. It's could be a TODO item.